packages/js/data/src/payment-settings/actions.ts

```diff
@@ -82,6 +82,20 @@ export function* togglePaymentGateway(
 	}
 }
 
+export function* attachPaymentExtensionSuggestion( url: string ) {
```

Comment:
Attach suggestions sounds a bit ambiguous to me. I guess naming may be improved, something like `storePaymentExtensionSuggestionActivation`.

Other option could be explaining what attach means in the comments across the changeset.

WDYT?

Reply:
Yes, it is ambiguous, but it is by design. Since we know so well, attachment is a broad concept, one that we will evolve and refine. Currently, it only means installing the plugin (not even activating it). At a later time, it might mean onboarding it. At an even later time, it might mean actually making transactions with it (the WooPayments definition of activation).

Reply:
I added explanations https://github.com/woocommerce/woocommerce/pull/56595/commits/b48d9fc5e69869a240e5a934b30f28a6db100499